app/controllers/grade_entry_forms_controller.rb

```diff
@@ -103,136 +73,17 @@ def update
-                                                       @students.total_pages,
-                                                       @sort_by)
-    session[:alpha_pagination_options] = @alpha_pagination_options
-    @alpha_category = @alpha_pagination_options.first
-  end
-
-  
-  # Handle pagination for grades table
-  # (The algorithm used to compute the alphabetical categories
-  # (alpha_paginate()) is
-  # found in grade_entry_form.rb.)
-  def g_table_paginate
-    @grade_entry_form = GradeEntryForm.find(params[:id])
-    @students, @students_total = handle_paginate_event(
-        G_TABLE_PARAMS,
-        {grade_entry_form: @grade_entry_form},
-        params)
-    # It is important to set the :per_page cookie during Ajax Request as well
-    c_per_page = cookie_per_page_name
-    if params[:per_page].present?
-      cookies[c_per_page] = params[:per_page]
-    elsif cookies[c_per_page].present?
-      params[:per_page] = cookies[c_per_page]
-    end
-    @current_page = params[:page]
-    @per_page = params[:per_page]
-    @filters = get_filters(G_TABLE_PARAMS)
-    @per_pages = G_TABLE_PARAMS[:per_pages]
-    @desc = params[:desc]
-    @filter = params[:filter]
-    c_sort_by = current_user.id.to_s + '_' +
-        @grade_entry_form.id.to_s + '_sort_by_grades'
-    if params[:sort_by].present?
-      @sort_by = params[:sort_by]
-    elsif cookies[c_sort_by].present?
-      params[:sort_by] = cookies[c_sort_by]
-    end
-
-    @sort_by = params[:sort_by]
-
-    # Only re-compute the alpha_pagination_options for the drop-down menu
-    # if the number of items per page has changed
-    if params[:update_alpha_pagination_options] == 'true'
-      all_students = get_filtered_items(
-          G_TABLE_PARAMS,
-          @filter,
-          @sort_by,
-          @desc)
-      @alpha_pagination_options = @grade_entry_form.alpha_paginate(
-          all_students,
-          @per_page,
-          @students.total_pages,
-          @sort_by)
-      @alpha_category = @alpha_pagination_options.first
-      session[:alpha_pagination_options] = @alpha_pagination_options
-    else
-      @alpha_pagination_options = session[:alpha_pagination_options]
-      @alpha_category = params[:alpha_category]
-    end
   end
 
-  # Cookie name
-  def cookie_per_page_name
-    "#{current_user.id}_#{@grade_entry_form.id}_per_page_sp"
-  end
-  
   # Update a grade in the table
   def update_grade
-    grade_entry_form = GradeEntryForm.find_by_id(params[:id])
+    grade_entry_form = GradeEntryForm.find(params[:id])
     @student_id = params[:student_id]
     @grade_entry_item_id = params[:grade_entry_item_id]
     updated_grade = params[:updated_grade]
 
-    grade_entry_student =
-      grade_entry_form.grade_entry_students.find_or_create_by_user_id(
-            @student_id)
+    grade_entry_student = grade_entry_form.grade_entry_students
+                                          .find_or_create_by_user_id(@student_id)
```

Comment:
Line is too long. [81/80]<br>Align the operands of an expression in an assignment spanning multiple lines.